components/sup/src/manager/service/package.rs

```diff
@@ -33,7 +33,7 @@ const PATH_KEY: &'static str = "PATH";
 static LOGKEY: &'static str = "PK";
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
-pub struct Env(HashMap<String, String>);
+pub struct Env(pub HashMap<String, String>);
```

Comment:
Scanning other changes I can't see why this was made public. Btw, this struct does implement `Deref<Target = HashMap<String, String>>` so you could call `deref()` to get at it's insides if you need it!

Reply:
It was actually the opposite; I have a `HashMap`, but I needed to turn it into an `Env` for testing, without having to mock out an installed package with a `RUNTIME_ENVIRONMENT` file.

Should have implemented `From`, I suppose.

Reply:
That's what I'd suggest, yep!